src/QueueManager.php

```diff
@@ -193,15 +192,14 @@ public static function engine(string $name): SimpleClient
      *      string 'default' if empty.
      * @return void
      */
-    public static function push($callable, array $args = [], array $options = []): void
+    public static function push(string $jobClass, array $args = [], array $options = []): void
     {
         // We can't use the callable type as it checks that the
         // [class, method] is statically callable which won't be true.
-        if (is_array($callable) && !class_exists($callable[0])) {
-            throw new InvalidArgumentException(
-                'Invalid callable provided. Please use either an array of `[classname, method]` or a string.'
-            );
+        if (! class_exists($jobClass)) {
```

Comment:
Should we allow short names too for consistency across the framework and use `App::className()` to get the FQCN?

Reply:
In that case we could do something like `QueueManager::push('Example')` and then resolve the FQCN with `App::className($jobClass, 'Job', 'Job')`. I guess that could work.

I'm not sure I'm in favor of that though. I like being able to click through to the job class in my IDE, so I'm not too keen on using a string that I need to interpret and then find. I think it would also be more confusing to beginners.

Reply:
Using App::className() doesn't prevent you from using FQCN. So it allows both type of usage.

Reply:
That is what I get for browsing code I don't know at 1 AM.

I wouldn't be opposed to using `App::className()`. I feel like that should be a separate pull request though since it would require more documentation and test changes.

Reply:
Please just update the code, the docs can be updated later :).

Reply:
Okay, updated code pushed.